mars/services/task/execution/ray/tests/test_ray_execution_backend.py

```diff
@@ -392,7 +373,16 @@ def pop(self, k, d=None):
     chunk_graph, subtask_graph = _gen_subtask_graph(t4)
     task = Task("mock_task", "mock_session", fuse_enabled=True)
     mock_config = RayExecutionConfig.from_execution_config(
-        {"backend": "ray", "ray": {"subtask_monitor_interval": 0}}
+        {
```

Comment:
Could you add a test which pass `num_cpus` in `execute`?

Reply:
There is a mock test in this PR.